WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -822,7 +824,7 @@ extension CollectOrderPaymentUseCaseError: CardPaymentErrorProtocol {
             return .dontRetry
         case .paymentGatewayNotFound:
             return .restart
-        case .unknownErrorRefreshingOrder, .couldNotRefreshOrder:
+        case .couldNotRefreshOrder, .orderTotalChanged:
```

Comment:
I don't think we should make this retryable, as the `Try again` button will always fail – at least in the existing app, probably in POS too though I wasn't able to recreate it there.

They need to go back to the order and refresh, then pay again.

I've updated this in 586568e